src/main/kotlin/app/batch/HBaseReader.kt

```diff
@@ -97,9 +98,14 @@ class HBaseReader constructor(private val connection: Connection) : ItemReader<S
             val scan = Scan().apply {
                 addColumn(columnFamily.toByteArray(), topicName.toByteArray())
             }
+
+            if (scanCacheSize.toInt() > 0) {
+                scan.caching = scanCacheSize.toInt()
+            }
+
+            logger.info("scan.caching: '${scan.caching}'.")
```

Comment:
Feels like a debug log, I'd make it say something more human readable like "Scan caching size set to [number]"

Reply:
@DanielChicot ☝️ ?